daemon/daemon.go

```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/docker/docker/daemon/exec"
 	"github.com/docker/docker/daemon/execdriver"
 	"github.com/docker/docker/daemon/execdriver/execdrivers"
-	"github.com/docker/docker/daemon/graphdriver"
 	_ "github.com/docker/docker/daemon/graphdriver/vfs" // register vfs
```

Comment:
Moving this to `layer_store.go` breaks `hack/make.sh cross` unfortunately. I've reverted it for now. 

The failures is:

```
# github.com/docker/docker/pkg/loopback
pkg/loopback/attach_loopback.go:21: undefined: LoNameSize
pkg/loopback/attach_loopback.go:21: invalid array bound LoNameSize
pkg/loopback/attach_loopback.go:22: undefined: LoNameSize
pkg/loopback/attach_loopback.go:22: invalid array bound LoNameSize
pkg/loopback/attach_loopback.go:119: undefined: loopInfo64
pkg/loopback/ioctl.go:11: undefined: LoopCtlGetFree
pkg/loopback/ioctl.go:19: undefined: LoopSetFd
pkg/loopback/ioctl.go:25: undefined: loopInfo64
pkg/loopback/ioctl.go:26: undefined: LoopSetStatus64
pkg/loopback/ioctl.go:39: undefined: loopInfo64
pkg/loopback/ioctl.go:26: too many errors
# github.com/docker/docker/pkg/devicemapper
pkg/devicemapper/devmapper.go:85: undefined: cdmTask
```

I think this change is large enough as it is now, we can move the registration import in another PR with the necessary changes to get `cross` working again. I'm not sure what's involved in that change.